ts/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import * as anchor from "@project-serum/anchor";
+import { BN } from "@project-serum/anchor";
+import { Connection, PublicKey } from "@solana/web3.js";
+
+export const CHAINLINK_AGGREGATOR_PROGRAM_ID = new PublicKey("HW3ipKzeeduJq6f1NqRCw4doknMeWkfrM4WxobtG3o5v");
+export const CHAINLINK_STORE_PROGRAM_ID = new PublicKey("CaH12fwNTKJAG8PxEvo9R96Zc2j8qNHZaFj8ZW49yZNT");
```

Comment:
Rebase and use the latest IDs

Reply:
I think these are current. They are live anyways. Lmk if not.